app/routes/reductions.js

```diff
@@ -13,25 +13,28 @@ module.exports = function (router) {
     if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
```

Comment:
Would be good to have an explicit route for edit reduction. As editing a reduction is essentially viewing. So, could we add that please?

Good for the customer to see correct urls and also for new devs coming on.